.changeset/witty-ducks-design.md

```diff
@@ -0,0 +1,8 @@
+---
+'@sap-cloud-sdk/e2e-tests': patch
+'@sap-cloud-sdk/odata-common': patch
+'@sap-cloud-sdk/odata-v2': patch
+'@sap-cloud-sdk/odata-v4': patch
+---
+
+[Compatibility Note] Fix deserializer so it works in batch requests which only contain functions or actions. This changes the type parameters of `ActionFunctionImportRequestBuilderBase`.
```

Comment:
[req] Please note that we are talking about deserializers and serializers, therefore (de-)serializers. Also, as this is a collection of functions we use the plural throughout the docs.

Reply:
got it, thanks for the explanation.